trade-finance-manager-api/src/v1/api.js

```diff
@@ -628,16 +628,14 @@ const updateGefFacility = async (facilityId, facilityUpdate) => {
   }
 };
 
-const queryDeals = async ({ queryParams, start = 0, pagesize = 0 }) => {
+const queryDeals = async ({ queryParams }) => {
```

Comment:
From the UI or here do we always set say pageSize already (on the queryParams) now? Wondering where we set these values now, related to my comment on the pageSize default in central API

Reply:
At the moment, we don't set the `pagesize` anywhere (so the UI/actual running application behaviour is unchanged) as the idea of this ticket was to introduce the _ability_ to paginate to the back-end without breaking any existing behaviour. We'll work out the best place to set (and the best value for) `pagesize` when we come to add the front-end changes as part of https://ukef-dtfs.atlassian.net/browse/DTFS2-6932